src/selfplay/tournament.cc

```diff
@@ -40,6 +40,7 @@ const char* kTrainingStr = "Write training data";
 const char* kNnBackendStr = "NN backend to use";
 const char* kNnBackendOptionsStr = "NN backend parameters";
 const char* kVerboseThinkingStr = "Show verbose thinking messages";
+const char* kResignPlaythroughStr = "What percentage of games ignore resign";
```

Comment:
It's not very consistent to have question rather than noun phrase. Maybe `Percentage of games with disabled resign`?

Reply:
Done